ansible/wait_for_vps.sh

```
@@ -0,0 +1,28 @@
+    
+    if ssh -o ConnectTimeout=10 -o BatchMode=yes -o StrictHostKeyChecking=no "$VPS_USER@$VPS_IP" 'echo "✅ VPS is back online!"' 2>/dev/null; then
+        echo "🎉 SUCCESS! VPS is reachable again!"
+        echo "🚀 You can now run the cleanup and redeployment."
+        exit 0
+    else
+        echo "❌ Still unreachable. Waiting $WAIT_TIME seconds..."
+        sleep $WAIT_TIME
+    fi
+done
+
+echo "⚠️  VPS still unreachable after $(($MAX_ATTEMPTS * $WAIT_TIME / 60)) minutes."
```

Shellcheck (0.10.0)

[style] $/${} is unnecessary on arithmetic variables.

(SC2004)